nx.json

```diff
@@ -0,0 +1,54 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
```

Comment:
This makes it so when main branch has new commits it caches the build to the cloud.  Any ci process or developer can then read from that cannonical remote cache

Reply:
thoughts on doing this without cloud for now?

Reply:
This actually isn't using cloud now.   IT doesn't start using cloud until you add an access token.   But it's set up to turn on cloud right away.